apps/sms/views/inbox/js/inbox.js

```diff
@@ -646,10 +581,9 @@ var InboxView = {
     }
 
     if (!empty) {
-      TimeHeaders.updateAll('header[data-time-update]');
+      this.list.setModel(this.listData);
```

Comment:
No that up to you. If you're model is empty, you can render something else inside or behind the list.